composer.json

```
@@ -14,24 +14,24 @@
     ],
 
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.0",
```

i think this should be 8.2, but I'm leaving as 8.0 for now

---

In line with our new policy to be explicit about the exact supported PHP range.
```suggestion
        "php": "8.1.* || 8.2.* || 8.3.* || 8.4.* ",
```